### DISCUSSION TITLE
这个项目为啥没有项目功能简介

### DISCUSSION THREAD

**Discussion by lizheng2018CC** [ZuodaoTech/everyone-can-use-english#288] [General]
看了好久，首页、issues，还有几个章节，都没找到该项目能用来做什么，也没有将软件界面展示在首页。但是鉴于该项目的star数，我觉得应该是我没找到，或者这是内部教学软件的开源版本？

**Comment by xiaolai** [comment]
原本只是一本书。

现在有软件了，但，目前算作是内部教学。
